styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java

```diff
@@ -457,48 +454,6 @@ void tickBackfillTriggerManager() {
     backfillTriggerManager.tick();
   }
 
-  private void initializeResources(Storage storage, ShardedCounter shardedCounter,
-                                   CounterSnapshotFactory counterSnapshotFactory,
-                                   TimeoutConfig timeoutConfig,
-                                   Supplier<Map<WorkflowId, Workflow>> workflowCache) {
-    try {
-      // Initialize resources
-      storage.resources().parallelStream().forEach(
-          resource -> {
-            try {
-              counterSnapshotFactory.create(resource.id());
-              storage.runInTransaction(tx -> {
-                shardedCounter.updateLimit(tx, resource.id(), resource.concurrency());
-                return null;
-              });
-            } catch (IOException e) {
-              LOG.error("Error creating a counter limit for {}", resource, e);
-              throw new RuntimeException(e);
-            }
-          });
-      LOG.info("Finished initializing resources");
-
-      // Sync resources usage
-      if (storage.config().resourcesSyncEnabled()) {
```

Comment:
We should remove this flag related code as well.

Reply:
And remove datastore configure after deploying.

Reply:
👍